openpathsampling/ensemble.py

```diff
@@ -48,12 +47,18 @@ def __init__(self, direction=None):
         self.prev_last_frame = None
         self.last_length = None
         self.direction = direction
-        self.contents = { }
+        self.contents = {}
 
     def bad_direction_error(self):
         raise RuntimeError("EnsembleCache.direction = " +
                            str(self.direction) + " invalid.") #nocover
 
+    def clear(self):
+        self.start_frame = None
+        self.prev_last_frame = None
+        self.last_length = None
+        self.contents = {}
+
```

Comment:
I'm not sure if this is ever/should ever be used. Note that the resetting mostly happens in the `check` method, with the option of forcing `reset=True`. I'm concerned that this method would lead someone to use it instead of `check(reset=True)` when the latter is the correct behavior.